cpp/cmake/thirdparty/get_cutlass.cmake

```
@@ -43,7 +43,7 @@ function(find_and_configure_cutlass)
   rapids_cpm_package_details(cutlass version repository tag shallow exclude)
 
   include("${rapids-cmake-dir}/cpm/detail/generate_patch_command.cmake")
-  rapids_cpm_generate_patch_command(cutlass ${version} patch_command)
+  rapids_cpm_generate_patch_command(cutlass ${version} patch_command False)
```

This will work, but is misleading. The last argument is the name of a variable to set.

```suggestion
  rapids_cpm_generate_patch_command(cutlass ${version} patch_command build_patch_only)
```